app/assets/stylesheets/custom.scss

```
@@ -4,6 +4,12 @@
 /* mixins, variables, etc. */
 $gray-medium-light: #eaeaea;
 
+@mixin box_sizing {
+  -moz-box-sizing:    border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing:         border-box;
```

```suggestion
  -moz-box-sizing: border-box;
  -webkit-box-sizing: border-box;
  box-sizing: border-box;
```
đẹp nhưng ko đúng nha =)) sai coding convention bé bì nhé